modules/test/playwright/pages/style-book-web/StyleBooksPage.ts

```diff
@@ -40,22 +54,49 @@ export class StyleBooksPage {
 	}
 
 	async deleteStyleBook(styleBookName: string) {
-		await this.page
-			.locator(
-				'input[name=_com_liferay_style_book_web_internal_portlet_StyleBookPortlet_keywords][type=search]'
-			)
-			.fill(styleBookName);
+		await this.searchStyleBook(styleBookName);
+
+		await this.page.getByLabel('More actions').click();
+		await this.page.getByRole('menuitem', {name: 'Delete'}).click();
+
+		await this.page.getByRole('button', {name: 'Delete'}).click();
+	}
 
-		await this.page.getByTitle('Search for', {exact: true}).click();
+	async editStyleBook(styleBookName: string) {
+		await this.searchStyleBook(styleBookName);
+
+		await this.page.getByLabel('More actions').click();
+		await this.page.getByRole('menuitem', {name: 'Edit'}).click();
+	}
+
+	
+	async searchStyleBook(styleBookName: string) {
+		await this.searchInput.fill(styleBookName);
+
+		await this.searchButton.click();
 
 		await expect(
 			this.page.getByText(`1 Result Found for "${styleBookName}"`)
 		).toBeVisible();
+	}
 
-		await this.page.getByLabel('More actions').click();
+	async updateTokenInputColor(label: string, colorHEX: string) {
+		const colorInput = this.page
+			.getByLabel(label)
+			.getByLabel('Color')
+			.locator('.layout__color-picker__input');
 
-		await this.page.getByRole('menuitem', {name: 'Delete'}).click();
+		await fillAndClickOutside(this.page, colorInput, colorHEX);
+	}
 
-		await this.page.getByRole('button', {name: 'Delete'}).click();
+	async publishStyleBook() {
+		await this.page.getByRole('button', {name: 'Publish'}).click();
+
+		await this.page
+			.getByRole('dialog')
+			.getByRole('button', {name: 'Publish'})
+			.click();
+
+		await waitForSuccessAlert(this.page, 'Success:Your request');
```

Comment:
No need to pass in the message, if you go to `utils/waitForSuccessAlert.ts` you'll see the message you are passing is the default value.